app/views/errors/submission_error.scala.html

```diff
@@ -0,0 +1,37 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import models.auth.User
+
+@()(implicit request: Request[_], appConfig: AppConfig, messages: Messages, user: User[_])
+
+@main_template(
+    title = messages("submissionError.title"),
+    appConfig = appConfig) {
+
+    @templates.heading(messages("submissionError.heading"))
```

Comment:
the prototype has `heading-xlarge` whilst the css on this title is `heading-large`. Can you correct it please